core/ld.py

```diff
@@ -429,6 +429,8 @@ def canonicalise(json_data: dict, include_security: bool = False) -> dict:
             "sensitive": "as:sensitive",
             "toot": "http://joinmastodon.org/ns#",
             "votersCount": "toot:votersCount",
+            "schema": "http://schema.org#",
+            "PropertyValue": "schema:PropertyValue",
```

Comment:
If you do this, you also need to change everywhere else that uses `http://schema.org#PropertyValue` as a lookup value (e.g. `fetch_actor`)

Reply:
Rather inline it.